src/ert/storage/local_storage.py

```diff
@@ -118,22 +118,20 @@ def _load_index(self) -> _Index:
 
     @no_type_check
     def _load_ensembles(self):
-        try:
-            ensembles = []
-            for ensemble_path in (self.path / "ensembles").iterdir():
+        ensembles = []
+        for ensemble_path in (self.path / "ensembles").iterdir():
+            try:
                 ensemble = self._load_ensemble(ensemble_path)
                 ensembles.append(ensemble)
-
-            # Make sure that the ensembles are sorted by name in reverse. Given
-            # multiple ensembles with a common name, iterating over the ensemble
-            # dictionary will yield the newest ensemble first.
-            ensembles = sorted(ensembles, key=lambda x: x.started_at, reverse=True)
-            return {
-                x.id: x
-                for x in sorted(ensembles, key=lambda x: x.started_at, reverse=True)
-            }
-        except FileNotFoundError:
-            return {}
+            except FileNotFoundError:
+                continue
+        # Make sure that the ensembles are sorted by name in reverse. Given
+        # multiple ensembles with a common name, iterating over the ensemble
+        # dictionary will yield the newest ensemble first.
+        ensembles = sorted(ensembles, key=lambda x: x.started_at, reverse=True)
```

Comment:
Kill this line

Reply:
Good point, did it in my original fix, but forgot here 🙈